### ISSUE TITLE
When importing swagger file, only first path param is transformed to Postman param

### ISSUE THREAD

**Issue by dor-denis** [a85/POSTMan-Chrome-Extension#2328]
1. **Postman Version:** 4.7.1
2. **App (Chrome app or Mac app):** Chrome app and Mac app
3. **OS details:** OS X El Capitan
4. **Is the Interceptor on and enabled in the app:** no
5. **Did you encounter this recently, or has this bug always been there:** always've been there
6. **Expected behaviour:** When I'm importing Swagger file, urls like `orders/{orderId}/shipments/{shipmentId}` transform to `orders/:orderId/shipments/:shipmentId`

Instead of this they are transformed to `orders/:orderId/shipments/{shipmentId}`. As you can see, only the first path param is transformed to editable Postman param with colon before it.

Steps to reproduce the problem:
1. Import Swagger file (I'm importing Swagger JSON from link) where there are paths with more than one path param (like `orders/{orderId}/shipments/{shipmentId}`)
2. See that the path was imported like `orders/:orderId/shipments/{shipmentId}`

**Comment by cviebrock** [comment]
I started noticing this when they fixed #2303 From version 4.6.3 (which didn't import Swagger at all) to 4.7.0 (which did, but only converted the first parameter).

And, when someone from Postman takes a look at the Swagger import, I'd suggest they also look into fixing https://github.com/postmanlabs/postman-app-support/issues/1782#issuecomment-231923643 ... where the default value for all the parameters is set to `{{orderId}}` (for example), instead of just leaving them empty.

**Comment by a85** [comment]
Yep. Looking into this.

**Comment by SamvelRaja** [comment]
The fix went in on 4.7.2 kindly update your app and check the same 😄

**Comment by dor-denis** [comment]
Just updated, the issue is still there for me :)

**Comment by SamvelRaja** [comment]
@dor-denis  Let me investigate further and get back to you

**Comment by vegetableman** [comment]
@dor-denis Could you check if the swagger file you are importing has `shipmentId` declared in the `parameters` array?

**Comment by dor-denis** [comment]
Yes it is included

``` json
{
    "/ds-orders/{orderId}/shipments/{shipmentId}": {
        "get": {
            "tags": [
                "dropshipping"
            ],
            "summary": "",
            "description": "",
            "parameters": [{
                "name": "expand",
                "in": "query",
                "description": "",
                "required": false,
                "type": "array",
                "items": {
                    "type": "string",
                    "enum": [
                        "packages",
                        "ds-order",
                        "sets",
                        "ds-items"
                    ]
                }
            }, {
                "name": "orderId",
                "in": "path",
                "description": "",
                "required": true,
                "type": "integer"
            }, {
                "name": "shipmentId",
                "in": "path",
                "description": "",
                "required": true,
                "type": "integer"
            }],
            "responses": {
                "200": {
                    "description": "",
                    "schema": {
                        "$ref": "#/definitions/Shipment"
                    }
                }
            },
            "security": [{
                "ppAuth": []
            }]
        }
    }
}
```

**Comment by vegetableman** [comment]
@dor-denis Thanks for sharing the data 👍 . We found the bug. The fix will be out in the next version.

**Comment by SamvelRaja** [comment]
@dor-denis @cviebrock We patched up a fix for this in our 4.8.0 version, kindly update your app and check the same from your end.

**Comment by cviebrock** [comment]
Works here.  Thanks!
